### PR TITLE
Lps 192275 | Add tests for deploy API Builder object definitions in all instances

### PR DESCRIPTION
**Pull request by yanan**
Background:
- Add tests for story ticket LPS-189447 Deploy API Builder object definitions in all instances

Test Plan:
background: Given a new virtual instance is created
- When I navigate to Control Panel > Objects
Then API Application, API Endpoint, API Filter, API Property, API Schema and API Sort are present
- When I navigate to /o/api in the virtual instance
Then the headless-builder/applications, headless-builder/endpoints, headless-build/filters, headless-builder/properties, headless-builder/schemas, headless-builder/shemas and headless-builder/sorts are loaded without errors
- When I create a new API application through Control Panel > Object > API Builder
Then the new API application is created successfully
- And Given I create a new API application through Control Panel > Object > API Builder
When I click Change Publication Status to publish the API application
Then the new API applicatin is published successfully without internal server error
- When with postAPIApplication to create a published API application
Then the API application is created successfully
And Then I can see the newly created API in API Builder

Jira Ticket:
- https://liferay.atlassian.net/browse/LPS-192275

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro [hunk 1]
```diff
@@ -12,11 +12,29 @@ definition {
 
 		Click(locator1 = "APIBuilder#CHANGE_PUBLICATION_STATUS_BUTTON");
 
-		if (!(isSet(check))) {
+		if (isSet(unpublish)) {
```
Comment (magjed4289) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro:15]: nice!

**Review by magjed4289 — APPROVED** [review]

**Review by Hijky — APPROVED** [review]

**Pull request closed without merging**